src/app.c

```diff
@@ -630,11 +637,16 @@ int on_command_line(GApplication* gapp, GApplicationCommandLine* cli, void* user
   shell_env = g_environ_setenv(shell_env, "TYM_ID", id_str, true);
   g_free(id_str);
 
+  char* cwd = option_get_str(option, "cwd");
```

Comment:
@endaaman

Sorry, I overlooked a small detail.

Since `g_application_command_line_get_cwd()` returns `const char*`, the variable probably needs to be changed to a `const char*` to keep the compiler happy.

Reply:
I noticed soon and fixed it. No problem!